data/src/config/server.rs

```diff
@@ -141,6 +148,56 @@ impl Sasl {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct WhoPollInterval(Duration);
+
+impl WhoPollInterval {
+    pub fn duration(&self) -> Duration {
+        self.0
+    }
+}
+
+impl Default for WhoPollInterval {
```

Comment:
Instead of newtyping both to implement Deserialize, you can tag the field with #[serde(deserialize_with = "")] and just define the single deser function to apply to both fields